fairing/response.go

```diff
@@ -0,0 +1,21 @@
+package fairing
+
+import (
+	"github.com/dannypsnl/rocket/response"
+)
+
+type ResponseHook func(*response.Response) *response.Response
+
+type Response struct {
```

Comment:
`Responder` might be better, `Response` would cause confusion with `response.Response`

Reply:
I don't think it will be confused, one is the `response.Response` and other is `fairing.Response`

Reply:
Maybe call `ResponseHandler`

Reply:
I think `ResponseDetractor` is okay.

Reply:
is Detractor is typo of Decorator?

Reply:
Oh shit www, yes that's typo